portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,22 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
 /**
- * Adds a random greeting to the page.
+ * Leads user to a selected project.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function chooseFeaturedProj() {
+  const urls =
+      ['https://github.com/georgezheng999/Depmap-Data-Scraping', 'https://github.mit.edu/MEDSL/primary-precincts', 'https://github.com/georgezheng999/Pac-Man', '', 'https://github.com/georgezheng999/Chess-Two-Player', 'https://github.com/georgezheng999/APCS-Spring-Projects'];
```

Comment:
Let's add some line breaks for long array like this to make it more readable.
https://google.github.io/styleguide/jsguide.html#formatting-array-literals

Reply:
Done.